maintenance_request_ifs_check/views/maintenance_views.xml

```diff
@@ -0,0 +1,80 @@
+<odoo>
+<data>
+    <!-- New tree view: stages - added in_progress boolean -->
+    <record id="hr_equipment_stage_view_tree" model="ir.ui.view">
+        <field name="name">equipment.stage.tree</field>
+        <field name="model">maintenance.stage</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_stage_view_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='done']" position="after">
+                <field name="ifs_checked"/>
+            </xpath>
+        </field>
+    </record>
+
+    <!-- New tree view: MaintenanceRequest - added ifs_checked -->
+    <record id="hr_equipment_request_view_tree" model="ir.ui.view">
+        <field name="name">equipment.request.tree</field>
+        <field name="model">maintenance.request</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_request_view_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='stage_id']" position="after">
+                <field name="ifs_checked" invisible="1"/>
+            </xpath>
+            <!-- decoration-bf - shows the line in BOLD
+                decoration-it - shows the line in ITALICS
+                decoration-danger - shows the line in LIGHT RED
+                decoration-info - shows the line in LIGHT BLUE
+                decoration-muted - shows the line in LIGHT GRAY
+                decoration-primary - shows the line in LIGHT PURPLE
+                decoration-success - shows the line in LIGHT GREEN
+                decoration-warning - shows the line in LIGHT BROWN -->
+            <xpath expr="//tree" position="attributes">
+                <attribute name='decoration-success'>ifs_checked</attribute>
+            </xpath>
+        </field>
+    </record>
+
+    <!-- New form view: MaintenanceRequest - added new IFS checks -->
+    <record id="hr_equipment_request_view_form" model="ir.ui.view">
+        <field name="model">maintenance.request</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_request_view_form"/>
+        <field name="priority" eval="100"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='description']" position="replace">
+                <notebook>
+                    <page string="Description">
+                        <field name="description" placeholder="Internal Note ......."/>
+                    </page>
+                    <page string="IFS checks">
```

Comment:
For better inheritance, add a name="page_ifs_checks" attribute

Reply:
Ok. I'll do it. Good idea.